modules/ROOT/pages/index.adoc

```diff
@@ -2,12 +2,13 @@
 :idprefix:
 :idseparator: -
 
-== Cache Enhancements
+== Write Scaling Enhancements
 
-Sync Gateway 2.6 supports enhancements to Sync Gateway’s cache management (channels, revisions) and supports additional configuration options to tune Sync Gateway cache settings.
-The smarter cache management techniques will allow the Sync Gateway to scale well with growth in the number of channels while the configuration options will give users better control over cache sizing.
+Sync Gateway 2.7 introduces enhancements which improve the write throughput.
+All Sync Gateway nodes can now participate in import processing, where as this was previously constrained to a single node.
+In addition, it guarantees the high availability of the import process and means that all Sync Gateway nodes can be deployed with the same configuration file.
 
-xref:deployment.adoc#channel-and-revision-cache[Read more]
+xref:shared-bucket-access.adoc[Read more]
 
 == Release Notes
```

Comment:
Please include following 
**Write Throughput Enhancements** 
We have improved the per-node throughput while processing writes that come in from Couchbase Lite clients [replicating](https://docs.couchbase.com/couchbase-lite/2.7/swift.html#starting-a-replication) with the Sync Gateway, as well as writes that are made via the Sync Gateway [REST API](https://docs.couchbase.com/sync-gateway/current/rest-api.html#/document).

**Import Processing Throughput** 
we have also significantly improved the performance of writes coming in from the Couchbase Server that are [imported](https://docs.couchbase.com/sync-gateway/2.6/config-properties.html#databases-foo_db-enable_shared_bucket_access) by the Sync Gateway . In addition to increasing the per-node import throughput, the enhancements also enable to scale linearly with the number of nodes. 

**High Availability of Imports**
In 2.7, import processing is automatically sharded across all the Sync Gateway nodes in a cluster. That way, even if the import node goes down, other nodes in the cluster can pick up the import shard. That way, import processing can continue uninterrupted with the other nodes in the cluster.

Reply:
@ibsoln  Please update the Whats New Page . You will need to add links